test/src/684-cvmfsbundles/main

```diff
@@ -0,0 +1,15 @@
+cvmfs_test_name="cvmfsbundles"
+cvmfs_test_autofs_on_startup=false
+cvmfs_test_suites="quick"
+
+
+cvmfs_run_test() {
+  logfile=$1
+
+  echo "*** Create a new repository"
+  create_empty_repo $CVMFS_TEST_REPO $CVMFS_TEST_USER || return 1
+
+
```

Comment:
We should already add code here to create .cvmfsbundles file, edit it, and remove it.  Also, we should already exercise failing malformed bundle files: a .cvmfsbundles directory and a .cvmfsbundles file in a subdirectory.